### PR TITLE
Allow importing steps and travelled distance from Withings as well;

### PR DESCRIPTION
**Pull request by bnjbvr**

### CODE REVIEW

**Comment by frankrousseau** [comment]
Thank you. It will require some time to review, because I have to check first that there it doesn't break my own setup ;)

File: server/konnectors/withings.coffee [hunk 1]
```diff
@@ -270,3 +303,52 @@ saveMeasures = (measures, callback) ->
...
+                notifContent = localization.t localizationKey, options
+            callback null, notifContent
+
+        callback()
+        return
+
+    console.log 'Fetching former activity measures...'
+    Steps.all (err, stepsMeasures) ->
+        return callback err if err
+        console.log 'Steps', stepsMeasures
```
Comment (frankrousseau) [inline server/konnectors/withings.coffee]: Useless log.

File: server/konnectors/withings.coffee [hunk 2]
```diff
@@ -270,3 +303,52 @@ saveMeasures = (measures, callback) ->
...
+            if not stepsHash[dateAsMom]?
+                stepMeasure = new Steps
+                stepMeasure.date = dateAsMom
+                stepMeasure.steps = steps
+                stepMeasure.vendor = 'Withings'
+                if newDistances[date]?
+                    stepMeasure.distance = newDistances[date].sum
+                stepsToSave.push stepMeasure
+
+        console.log "Found #{stepsToSave.length} new steps measures to save!"
```
Comment (frankrousseau) [inline server/konnectors/withings.coffee]: Use printit to log information.

File: server/konnectors/withings.coffee [hunk 3]
```diff
@@ -270,3 +303,52 @@ saveMeasures = (measures, callback) ->
...
+
+            console.log 'Steps measures saved.'
+            notifContent = null
+            if stepsToSave.length
+                localizationKey = 'notification withings'
+                options = smart_count: stepsToSave.length
+                notifContent = localization.t localizationKey, options
+            callback null, notifContent
+
+        callback()
```
Comment (frankrousseau) [inline server/konnectors/withings.coffee]: It look like callback is called twice.

File: server/konnectors/withings.coffee (hunk 1 above)
Comment (bnjbvr) [inline server/konnectors/withings.coffee]: Removed this one and the same in the body measures (there was one such log for the scales)

File: server/konnectors/withings.coffee (hunk 3 above)
Comment (bnjbvr) [inline server/konnectors/withings.coffee]: Good catch, thanks!

**Comment by bnjbvr** [comment]
I've just updated the PR, take your time for reviewing it, it's not in a hurry ;)

**Comment by frankrousseau** [comment]
Thank you for your contribution @bnjbvr !

**Comment by bnjbvr** [comment]
Thanks for merging! I've just realized I forgot to `git add` my modifications before committing and pushing back onto my server, and so your review comments weren't addressed by my commit. Sorry about that!

**Comment by frankrousseau** [comment]
np

**Pull request merged**